src/pages/RegistrationPage.js

```diff
@@ -110,19 +95,14 @@ class RegistrationPage extends React.Component {
 
   static validate(values, submitted) {
     let registrationFields = RegistrationFields
-    deleteEmptyArrayKeys(values)
+    //deleteEmptyArrayKeys(values) //TODO: What is this
```

Comment:
If the Checkbox or Radio buttons the original form used to have where empty(not selected) this function will remove the key from the values been passed to the validator.

Reply:
Thats what I thought it did, we actually need the field to be there so for future checks. What was the reason we wanted to remove the key completely?